pkg/cover/html.go

```diff
@@ -353,7 +353,7 @@ func fileContents(file *file, lines [][]byte, haveProgs bool) string {
 			if end == start {
 				continue
 			}
-			chunk := htmlReplacer.Replace(string(ln[start:end]))
+			chunk := htmlReplacer.Replace(fmt.Sprintf("%v %v", i, string(ln[start:end])))
```

Comment:
If I am reading this code correctly, we can write several chunks per line (e.g. not-covered/covered/not-covered), are you are prepending line to each chunk... does it mean we will get multiple lines numbers per line?

Reply:
No, it won't, as there is only one tr and one td here.
I verified it in the cover page too, only one number before each line.

Reply:
1. I'm seeing this error when I enabled cover for gVisor:
machine check failed: coverage is not supported (debugfs is not enabled or not mounted)

and no coverage shown if set `cover: true`.
![image](https://user-images.githubusercontent.com/77719320/122341308-eb419880-cf75-11eb-9fad-38e3f13649bd.png)


2. A snapshot for cover page of linux target device.
![image](https://user-images.githubusercontent.com/77719320/122341165-c0efdb00-cf75-11eb-9774-05fc4012ba0d.png)

Reply:
> I'm seeing this error when I enabled cover for gVisor:

You need these additional flags when building gvisor to get coverage support:
https://github.com/google/syzkaller/blob/master/pkg/build/gvisor.go#L39-L61

Reply:
I had used the command below to build gvisor, but still same error: ```machine check failed: coverage is not supported (debugfs is not enabled or not mounted)```, I actually has mounted debugfs.

```
bazel build  --verbose_failures --collect_code_coverage --instrumentation_filter=//pkg/…,-//pkg/sentry/platform,-//pkg/ring0,-//pkg/coverage:coverage //runsc:runsc
```

But anyhow, I will update the PR to have single line number.

Reply:
updated the PR with new patch.

Reply:
Hummm.... I don't know I did not run gvisor manually for some time.
Maybe you can just look at one of syzbot coverage reports for gvisor:
https://syzkaller.appspot.com/gvisor
For instances with -cover you should be able to easily find lines that has both covered and uncovered parts.

Reply:
Thanks. I think I got the idea behind, it just like lcov report which has several branches on the same line and some covered, some not.